sdks/go/pkg/beam/transforms/sql/sql_test.go

```diff
@@ -0,0 +1,105 @@
+package sql
+
+import (
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/transforms/sql/sqlx"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+func TestOptions_Add(t *testing.T) {
+	o := options{}
+	str := "this is a string"
+	bytes := []byte{1, 2, 3, 4}
+	testOpt := sqlx.Option{
+		Urn:     str,
+		Payload: bytes,
+	}
+	tests := []struct {
+		opt sqlx.Option
+	}{
+		{
+			opt: testOpt,
+		},
+	}
+
+	for _, test := range tests {
+		o.Add(test.opt)
+		if o.customs == nil || !reflect.DeepEqual(o.customs[len(o.customs)-1], test.opt) {
+			t.Errorf("Add failed.")
```

Comment:
This isn't a great error message. If there were additional test cases, it wouldn't be possible to know which one is which. A good test error identifies the input and the function under test, so the failure output avoids a reader from needing to hunt for the important details.

See https://google.github.io/styleguide/go/decisions#identify-the-input and https://google.github.io/styleguide/go/decisions#identify-the-function for tips.

Reply:
In my last commit, I changed it to 
```go
t.Errorf("The method options.Add(%v) did not successfully perform the add operation", test.opt)
```
I attempted to use the form `YourFunc(%v) = %v, want %v` but found out that the method I'm testing does not have a return value.